dist/carousel/ds4/carousel.css

```diff
@@ -214,7 +214,7 @@ span.carousel__container {
     -webkit-scroll-snap-type: x mandatory;
     -ms-scroll-snap-type: mandatory;
     -ms-scroll-snap-type: x mandatory;
-    scroll-snap-type: proximity;
+    scroll-snap-type: mandatory;
```

Comment:
Carousel changes should not be included as part of this PR.

Reply:
ah whoops.. forgot to git clean